tests/validation/cattlevalidationtest/core/test_lb.py

```diff
@@ -41,13 +41,18 @@ def fin():
                     to_delete.append(i)
             except AttributeError:
                 pass
+        delete_all(client, to_delete)
```

Comment:
I think it would be more clear if we introduce a new datastructure for each object-to-remove instead of explicitly nulling the same one